packages/indexer/src/testing/indexer.ts

```diff
@@ -0,0 +1,21 @@
+import { MockStream } from "@apibara/protocol/testing";
+import { createIndexer, defineIndexer } from "../indexer";
+
+export const mockIndexer = createIndexer(
+  defineIndexer(MockStream)({
+    streamUrl: "https://sepolia.ethereum.a5a.ch",
+    finality: "accepted",
+    filter: {},
+    transform({ block: { block_number } }) {
+      if (!block_number) return [];
+
+      const res: MockRet[] = [{ block_number: Number(block_number) }];
+
+      return res;
```

Comment:
```suggestion
      retur [{ block_number: Number(block_number) }];
```

Add `as MockRet[]` if needed.